### PR TITLE
Support settings roles by default on authentication

### PR DESCRIPTION
**Pull request by Sebobo**

### CODE REVIEW

File: Configuration/Settings.yaml [hunk 1]
```diff
@@ -1,4 +1,11 @@
 TYPO3:
   LDAP:
     # Allow login based on cached credentials if the directory service is offline
-    allowStandinAuthentication: FALSE
\ No newline at end of file
+    allowStandinAuthentication: FALSE
+    roles:
+      default:
```
Comment (radmiraal) [inline Configuration/Settings.yaml]: shouldn't this be an empty array too?

File: Classes/TYPO3/LDAP/Security/Authentication/Provider/LDAPProvider.php [hunk 2]
```diff
@@ -173,6 +215,63 @@ protected function updateParty(Account $account, array $ldapSearchResult) {
...
+			}
+			$eelContext = new Context($contextVariables);
+
+			foreach ($this->rolesConfiguration['default'] as $roleIdentifier) {
+				$role = $this->policyService->getRole($roleIdentifier);
+				$account->addRole($role);
+			}
+
+			$dn = $this->eelEvaluator->evaluate($this->partyConfiguration['dn'], $eelContext);
+
```
Comment (radmiraal) [inline Classes/TYPO3/LDAP/Security/Authentication/Provider/LDAPProvider.php]: if you're on fixing that other comment about the empty array in the config file I'd remove this empty line (236) too

**Comment by Sebobo** [comment]
Hey @radmiraal can you check this PR again? 
I accidentally added two more changes ;)

**Comment by radmiraal** [comment]
thnx ;) I need to do an LDAP / Active Directory SSO setup within the next 2 weeks, will give the change a spin while doing so

**Comment by radmiraal** [comment]
Found a few minor tweaks which I'll push in another change, but this definitely works / looks good ;-) thanks!

**Pull request merged**